tests/releases/testdata/missing_reference.txt

```diff
@@ -0,0 +1,7 @@
+bigquery.cnrm.cloud.google.com/v1beta1, Kind=BigQueryRoutine
```

Comment:
Is the goal for this empty to be empty? Do we want to try to hold a line of not adding to this file? If so it would be good to have a comment which explained that.

Reply:
We want to add a check to have a clean "missing_reference" before each release. It would be a presubmit check for release tag PRs (or some other auto-check)